### DISCUSSION TITLE
How can solve this problem?

### DISCUSSION THREAD

**Discussion by Ankai1e** [RedShyGuy/Vapecord-ACNL-Plugin#92] [Q&A]
Running acnl loads the plugin infinitely in the upper left corner. Also, the plugin is not running and the game speed has slowed down.
![20211208_012635](https://user-images.githubusercontent.com/95704506/145067781-0ac74fca-d5c1-42c8-83f0-84ddedbf2b79.jpg)

**Comment by RedShyGuy** [comment]
Update your game

**Reply by Ankai1e** [reply]
Thanks for answer.
Thanks to I was able to solve this problem.
